### PR TITLE
Fixed web entity detection API and added tests

### PR DESCRIPTION
**Pull request by telescopic**
* Fixed web entity detection API to handle url's instead of local images
* Added simple tests for web entity detection

### CODE REVIEW

File: tests/summmarizer/test_image_description_retreiver.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import json
+from unittest.mock import Mock, patch
+
+from classifier_and_summarizer.summarizer.web_entity_detection import \
+    ImageDescriptionRetriever
+
```
Comment (tmadhuri) [inline tests/summmarizer/test_image_description_retreiver.py:6]: Add tests to verify the following as well:
- the request for web entity detection is formatted correctly
- error with correct description is returned when the server responds with an error code != 200

File: tests/summmarizer/test_image_description_retreiver.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        ["https://tinyurl.com/y7how2rj"])[0]
+    assert 'sundar pichai' in reponse_for_image_url_1['label']
+
+    reponse_for_image_url_2 = image_describer.get_description_for_images(
+        ["https://tinyurl.com/y9bvoehm"])[0]
+    assert 'larry page' in reponse_for_image_url_2['label']
+
+    reponse_for_image_url_3 = image_describer.get_description_for_images(
+        ["https://tinyurl.com/y9t35t3z"])[0]
+    assert 'sergey brin' in reponse_for_image_url_3['label']
```
Comment (tmadhuri) [inline tests/summmarizer/test_image_description_retreiver.py:66]: verify for 'entities' as well

Comment (telescopic) [reply tests/summmarizer/test_image_description_retreiver.py:66]: Updated Accordingly,

File: tests/summmarizer/test_image_description_retreiver.py (hunk 1 above)
Comment (telescopic) [reply tests/summmarizer/test_image_description_retreiver.py:8]: Added the tests mentioned.

File: tests/summmarizer/test_image_description_retreiver.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            }
+        ]
+    })
+    return Mock(status_code=status_code, content=response)
+
+
+@patch(
+    'classifier_and_summarizer.summarizer.web_entity_detection.requests.post',
+    side_effect=mocked_requests_post)
+def test_response_format(mocked_post):
```
Comment (tmadhuri) [inline tests/summmarizer/test_image_description_retreiver.py:55]: We don't need to verify the response from the server. I meant that we need to verify the request we send to the server.

Comment (telescopic) [reply tests/summmarizer/test_image_description_retreiver.py]: Apologies. Updated accordingly!

**Pull request merged**